certified-connectors/Verified/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2681 @@
+            }
+          }
+        },
+        "x-ms-visibility": "advanced"
+      }
+    },
+    "/envelopes/{envelope-id}/publish-status": {
+      "put": {
+        "operationId": "PutEnvelopesEnvelopeIdPublishStatus",
+        "summary": "Update publish status for envelope",
+        "description": "Updates the publish status of a given envelope",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "published": {
+                  "type": "boolean",
+                  "title": "Published",
+                  "enum": [
+                    true,
+                    false
+                  ],
+                  "x-ms-visibility": "important"
+                }
+              },
+              "example": {
+                "published": true
+              },
+              "x-ms-visibility": "important",
+              "required": [
+                "published"
+              ]
+            },
+            "required": true,
+            "x-ms-visibility": "important"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "",
+            "schema": {
+              "type": "object"
+            }
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/envelopes/{envelope-id}/documents/{document-id}/templates/{template-id}/user-data": {
+      "post": {
+        "operationId": "PostEnvelopesEnvelopeIdDocumentsDocumentIdTemplatesTemplateIdUserData",
+        "summary": "Generate Document from Template",
+        "description": "Generates a document from a preconfigured template.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          },
+          {
+            "$ref": "#/parameters/DocumentId"
+          },
+          {
+            "name": "template-id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Template Id",
+            "description": "The Id of the template which should be used",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object"
+            },
+            "required": true,
+            "x-ms-visibility": "important"
+          }
+        ],
+        "responses": {
```

Comment:
@BenediktBergmann Thanks for sharing the API documentation, You can keep the empty schema for now. 
But have a valid description value for the response code. But this may get flagged during the final review during prepping your connector production deployment. I will let you know with a resolution if it gets blocked.